### ISSUE TITLE
Zignature have issue with Bulleted List

### ISSUE THREAD

**Issue by Videoso** [zendesklabs/zignatures#8]
Hi Team!

I try your APP and I like it.

An Agent use this bulleted list feature in Zendesk.

If we mark 3 entrys with Bulleted points, then Zignature APP will add 3-8 more also before signature.

May anyone know a solution here?

Best Regards,

Tobias

**Comment by Sarapulov** [comment]
thanks Tobias!
Can you please add a screenshot and example of signature that is saved in app settings?

**Comment by Videoso** [comment]
Hi,

sure, please have a look here:

Here is Signature:

{{dc.best_regards}}<br>{{current_user.name}}<img class='logo-image' src='http://fs1.directupload.net/images/180103/bxjgrvvp.png' width='101' height='40' alt=Zyxel Support Campus'>

Please ignore last HTML line, that´s testing purpose and not releated to issue.

Many thanks for fast feedback.

Best Regards,

Tobias

[cid:image001.png@01D388A0.D1C4D900]

**Comment by Videoso** [comment]
![screenshot](https://user-images.githubusercontent.com/35229888/34678732-44a5ffee-f494-11e7-8f44-3068295df7c7.png)

Add Screenshot again.

**Comment by Sarapulov** [comment]
Thanks!
I  am full of travel plans this week. So may be challenging for me to check in very short term. 
However, it's flagged for me to investigate. 

Very first impression is that it has something to do with comment editor. Zignatures simply appends the content to the bottom of existing comment following its styles and formatting. From the screenshot appears that comment was in "bullet point" mode. So this formatting was inherited. Perhaps try to complete the list and the save the comment. OR (more hacky way) pre pend to the signature HTML closing tags for bulleted  list. I think it could be </li></ul> these may be ignored when no list used and respected when it used. Let me know if any of these work? I will perform proper check later.

**Comment by Videoso** [comment]
Hi Sarapulov,

so far, both things fail.

Even when I strictly unselect this Bullet point mode and type something else, then click Submit and it put this Zignatures Code in the bottom, the bottom will be flagged with this bullet point.

Also, LI / UL will fail.

So I´ll wait for another news from you.

Happy traveling!

![2](https://user-images.githubusercontent.com/35229888/34681449-38246028-f49c-11e7-8b98-301517ffcc73.png)

**Comment by Sarapulov** [comment]
Thanks for trying!

**Comment by Videoso** [comment]
Update some more testing:

When I use /li or /ul in Zignature Config, it fails.

When I use Zendesk Rich Text Editor and put [/ul] in the bottom, then the signature is fine.

However, when disabling Zignature and put the Dynamic content manually in Rich Text Editor or insert from Zendesk it´s also fine.

Thanks.

Tobias

**Comment by Sarapulov** [comment]
Hi Tobias,

Thanks again for reporting your findings!

I took your signature HTML example and fix it (quote was missing for `alt` attribute in `img` tag).
I had to replace Dynamic Content with a text.
Added `<br>` at the very beginning to shift signature to the new line.
Below is what I used for the Zignature app:

```
<br>Best regards<br>{{current_user.name}}<br><img class="logo-image" src="http://fs1.directupload.net/images/180103/bxjgrvvp.png" width="101" height="40" alt="Zyxel Support Campus" />
```

I am able to replicate the issue when bulleted list left open at the moment of saving a ticket (see below):

![](https://cl.ly/3a0d2e2L2y0Z/Screen%20Recording%202018-01-13%20at%2005.28%20PM.gif)

However, as soon as I close bulleted list things started to work as expected (see below):

![](https://cl.ly/1b0g3i0Y2n1R/Screen%20Recording%202018-01-13%20at%2005.30%20PM.gif)

The behaviour seems logical to me as Zignature app inherit bulleted list formatting when it stays open. This seems like generally not a best practice to save comment with opened bulleted list. I suggest your agents will always make it close before sending a ticket. Would that be an answer here?
